build-config/resources/Ballerina.toml

```diff
@@ -5,6 +5,7 @@ version = "@toml.version@"
 authors = ["Ballerina"]
 keywords = ["FTP", "SFTP", "remote file", "file transfer", "client", "service"]
 repository = "https://github.com/ballerina-platform/module-ballerina-ftp"
+icon = "resources/icon.png"
```

Comment:
Lets have it at the ballerina project root

Reply:
Moved.